src/components/SortableTable.js

```diff
@@ -0,0 +1,292 @@
+          marginRight: '20px',
+          marginLeft: '25px'
+        },
+        displayedRows: {
+          fontFamily: 'Montserrat',
+          color: '#626262',
+          fontSize: '12px',
+        }
+      }
+    },
+    MuiTableCell: {
+      styleOverrides: {
+        root: {
+          fontFamily: 'Montserrat',
+          color: '#000',
+          fontSize: '14px',
+          fontWeight: '400',
+          padding: '7px 20px 7px 20px'
+        }
+      }
+    },
+  }
+});
+
+function descendingComparator(a, b, orderBy) {
+  if (b[orderBy] < a[orderBy]) {
+    return -1;
+  }
+  if (b[orderBy] > a[orderBy]) {
+    return 1;
+  }
+  return 0;
+}
+
+function getComparator(order, orderBy) {
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+function stableSort(array, comparator) {
+  const stabilizedThis = array.map((el, index) => [el, index]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) {
+      return order;
+    }
+    return a[1] - b[1];
+  });
+  return stabilizedThis.map((el) => el[0]);
+}
+
+function EnhancedTableHead(props) {
+  const {
+    order,
+    orderBy,
+    onRequestSort,
+    headCells
+  } = props;
+  const createSortHandler = (property) => (event) => {
+    onRequestSort(event, property);
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <TableHead>
+        <TableRow>
+          {headCells.map((headCell) => (
+            <TableCell
+              key={headCell.id}
+              align={headCell.numeric ? 'right' : 'left'}
+              padding={headCell.disablePadding ? 'none' : 'normal'}
+              sortDirection={orderBy === headCell.id ? order : false}
+              sx={{
+                lineHeight: 'normal',
+                fontWeight: '600',
+                padding: '10px'
+              }}
+            >
+              <TableSortLabel
+                active={orderBy === headCell.id}
+                direction={orderBy === headCell.id ? order : 'asc'}
+                onClick={createSortHandler(headCell.id)}
+                sx={{
+                  fontSize: '16px',
+                  fontWeight: '400'
+                }}
+              >
+                {headCell.label}
+                {orderBy === headCell.id ? (
+                  <Box component='span' sx={visuallyHidden}>
+                    {order === 'desc' ? 'sorted descending' : 'sorted ascending'}
+                  </Box>
+                ) : null}
+              </TableSortLabel>
+            </TableCell>
+          ))}
+        </TableRow>
+      </TableHead>
+    </ThemeProvider>
+  );
+}
+
+EnhancedTableHead.propTypes = {
+  onRequestSort: PropTypes.func.isRequired,
+  order: PropTypes.oneOf(['asc', 'desc']).isRequired,
+  orderBy: PropTypes.string.isRequired,
+};
+
+export default function SortableTable(props) {
+
+  const {
+    rows,
+    headCells
+  } = props;
```

Comment:
I think this component would be more useful if a predefined `theme` were passed in as a prop from the caller.
As a side note, we do have another `theme.js` class which can seem confusing. I'm wondering if `SortableTable` could be it's own package level (i.e. directory) component with `EnhancedTableHead` and `Themes` as separate components in that package.

Reply:
I can implement it that way! I think the SortableTable file is way too crowded.